src/components/SelectSearch/styled.ts

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import type { CSSProperties, ReactNode } from 'react'
+
+import { ChevronDown } from '@amsterdam/asc-assets'
+import { themeColor, themeSpacing } from '@amsterdam/asc-ui'
+import { INPUT_SIZE } from '@amsterdam/asc-ui/lib/components/shared/constants'
+import styled, { css } from 'styled-components'
+
+export interface Props {
+  id?: string
```

Comment:
Why are there no props obligatory? And now that I'm looking at it, why is this in the styled.ts?

Reply:
i removed them, copied al the options from another Select Component, but not needed for this usecase!